SoftLayer/CLI/vlan/edit.py

```diff
@@ -22,6 +23,9 @@
 def cli(env, identifier, name, note, tags):
     """Edit a vlan's details."""
 
+    if not (name or note or tags):
+        raise exceptions.CLIAbort("Is required Name or Note or Tags")
```

Comment:
"Name, Note or Tags is required" would be a more natural sounding sentence. But for this case, I'd prefer the following.

```suggestion
        raise exceptions.CLIAbort("At least one option is required")
```

Reply:
Done